src/lib/src/component/navigation/navigation.component.ts

```diff
@@ -114,9 +116,19 @@ export class NavigationComponent implements OnInit {
   }
 
   run(): void {
-    this.executionService.execute(this.uiState.selectedElement.path).then(response => {
-        if (response.status === 200) {
-          this.uiState.selectedElement.state = ElementState.Running;
+    let selectedElement = this.uiState.selectedElement;
+    this.executionService.execute(selectedElement.path).then(response => {
+        if (response.status === this.HTTP_STATUS_CREATED) {
+          selectedElement.state = ElementState.Running;
+          this.notification = `Execution of ${selectedElement.name} has been started.`;
```

Comment:
does the name include the extension '.tcl'? If so, please remove it. I'd prefer putting the name into quotes.